src/plugins/discover/public/types.ts

```diff
@@ -38,3 +38,5 @@ export interface DataTableRecord {
    */
   isAnchor?: boolean;
 }
+
+export type SortOrder = [string, string];
```

Comment:
We could replace it with `SortPairArr` type, no?

Reply:
yes we could further simplify 👍 , less types, more fun